### PR TITLE
Add 0.106 release notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,4 +1,7 @@
 * 0.106
```
Comment (cberner) [inline CHANGES]: I think this should be "0.107"

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,12 @@
 Airlift 0.106
```
Comment (cberner) [inline NEWS]: same

**Comment by dain** [comment]
fixed

File: NEWS [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
 
+We have replaced use of SLF4J in the logging backend with a custom logging
+implementation directly connected to Java util logging.  This makes management
+of logging dependencies as the core codebase no longer depends directly on any
```
Comment (cberner) [inline NEWS]: missing word? "This makes management of logging dependencies as... [sic]"

**Comment by cberner** [comment]
Looks good, besides the one comment

**Pull request merged**